migrations/20200304201158_add_user_profile_columns.js

```diff
@@ -0,0 +1,14 @@
+
+exports.up = function(knex) {
+  return knex.schema.table('users', table => {
+    table.string('preferedClan')
```

Comment:
🚨 `preferred` has double **rr**. After updating here, don't forget to also update on the `down` export.

🚨 References to clan should be restricted by referencing the `clans` table, and referencing by clan id. [Check how we did it here](
https://github.com/l5r-discord-league/discord-league/blob/master/migrations/20200207012314_create_matches.js#L31-L32).

By using references we safeguard ourselves against mistyping something and entering the wrong clan name, and keep everything tidier so we can do statistical work later. It also makes it easier to attach information to specific clans later, if a use for it comes up (_like having a boolean column isEvil, so we can make keep it clear that scorp is not evil_)